SingularityUI/app/views/requestForm.jsx

```diff
@@ -0,0 +1,23 @@
+import React from 'react';
+import ReactDOM from 'react-dom';
+
+import View from './view';
+
+import { Provider } from 'react-redux';
+
+import RequestFormPage from '../components/requestForm/RequestForm'
+
+
+class RequestFormView extends View {
+  initialize({store, editing}) {
+    this.store = store;
+    this.editing = editing;
```

Comment:
my advice would be:
- Create a new action that clears out the request
- Update the controller to trigger this action instead of short circuiting to `Promise.resolve()`
- Rip out instances of `editing` once this is confirmed working